tests/reference/asr-test_unsupported_type-0d813dd.stderr

```diff
@@ -0,0 +1,5 @@
+semantic error: Unsupported type annotation: i128
+ --> tests/errors/test_unsupported_type.py:2:5
+  |
+2 |     i: i128
+  |     ^^^^^^^ 
```

Comment:
Can you try to improve the message like this?
```suggestion
2 |     i: i128
  |        ^^^^ 
```

Reply:
I think that can be done in a subsequent PR. This PR fixes the main problem to raise an error, which now works.